pyta.py

```diff
@@ -26,6 +27,9 @@ def check(module_name):
     The name of the module should be passed in as a string,
     without a file extension (.py).
     """
+    if sys.version_info<(3,3,0):
+        sys.stderr.write("You need python 3.3 or later to run this script\n")
```

Comment:
Please use single-quotes; also, only allow Python version 3.4+. And also, please capitalize "Python."